Update Settings Pages (#2242)
hashicorp/terraform-website: content/cloud-docs/users-teams-organizations/users.mdx

# Users

> User accounts belong to individual people. Each user can be part of one or more [teams](/cloud-docs/users-teams-organizations/teams), which are granted permissions on workspaces within an organization. A user can be a member of multiple [organizations][].

Reviewer: ```suggestion
User accounts belong to individual people. Each user can be part of one or more [teams](/cloud-docs/users-teams-organizations/teams), which are granted permissions on workspaces within an organization. A user can be a member of multiple [organizations][].
```

## API
Use the [Account API](/cloud-docs/api-docs/account) to get account details, update account information, and change your password.

## Creating an Account

Creating an account requires a username, an email address, and a password. To use Terraform Cloud, you must create an account through one of the following methods:

- **Invitation Email:** When a user sends you an invitation to join an existing Terraform Cloud organization, the email includes a sign-up link. After you create an account, you will automatically join that organization and can begin using Terraform Cloud.

> - **Sign-Up Page:** For Terraform Cloud, go to [https://app.terraform.io/signup/account](https://app.terraform.io/signup/account). For Terraform Enterprise, go to `https://<TFE HOSTNAME>/signup/account`. After you create an account, you will not belong to any organizations. To begin using Terraform Cloud, you can either [create an organization](/cloud-docs/users-teams-organizations/organizations#creating-organizations) or ask an organization owner to send you an invitation email.

Reviewer 2: ```suggestion
- **Sign-Up Page:** For Terraform Cloud, go to [https://app.terraform.io/signup/account](https://app.terraform.io/signup/account). For Terraform Enterprise, go to `https://<TFE HOSTNAME>/signup/account`. After you create an account, you will not belong to any organizations. To begin using Terraform Cloud, you can either [create an organization](/cloud-docs/users-teams-organizations/organizations#creating-organizations) or ask an organization owner to send you an invitation email to join their organization.
```

> - **Sign-Up Page:** For Terraform Cloud, go to [https://app.terraform.io/signup/account](https://app.terraform.io/signup/account). For Terraform Enterprise, go to `https://<TFE HOSTNAME>/signup/account`. After you create an account, you will not belong to any organizations. To begin using Terraform Cloud, you can either [create an organization](/cloud-docs/users-teams-organizations/organizations#creating-organizations) or ask an organization owner to send you an invitation email to join their organization.

Reviewer: ```suggestion
- **Sign-Up Page:** For Terraform Cloud, go to [`https://app.terraform.io/signup/account`](https://app.terraform.io/signup/account). For Terraform Enterprise, go to `https://<TFE HOSTNAME>/signup/account`. After you create an account, you will not belong to any organizations. To begin using Terraform Cloud, you can either [create an organization](/cloud-docs/users-teams-organizations/organizations#creating-organizations) or ask an organization owner to send you an invitation email to join their organization.
```

## Joining Organizations and Teams

> An organization owner must [invite you to join an organization](/cloud-docs/users-teams-organizations/organizations#users). After you join, the organization owner must also [add you to one or more teams](/cloud-docs/users-teams-organizations/teams#managing-team-membership).

Reviewer 2: ```suggestion
An organization owner must [invite you to join their organization](/cloud-docs/users-teams-organizations/organizations#users). After you join, the organization owner must also [add you to one or more teams](/cloud-docs/users-teams-organizations/teams#managing-team-membership).
```

I know this is obvious, but I feel that this change gets rid of a small understanding loophole where users might believe that an invite from any org owner could get them access to other orgs which aren't associated with the owner who invited them.

Author: yes! thank you!

[permissions-citation]: #intentionally-unused---keep-for-maintainers

Terraform Cloud sends user invitations by email. If the invited email address matches an existing Terraform Cloud account, the invitee can join the organization with that account. Otherwise, they must create a new account to work with that organization.

## Site Admin Permissions

> On Terraform Enterprise instances, some user accounts have a special site admin permission that allows them to administrate the entire instance.

Reviewer: ```suggestion
On Terraform Enterprise instances, some user accounts have a special site admin permission that allows them to administer the entire instance.
```

> On Terraform Enterprise instances, some user accounts have a special site admin permission that allows them to administrate the entire instance.

Author: thank you!! Fixed - judith also flagged it :)

You have an SSO identity for every SSO-enabled Terraform Cloud organization. Terraform Cloud links each SSO identity to a single Terraform Cloud user account. This link determines which account you can use to access each organization.

### Tokens

Click **Tokens** in the sidebar to create, manage, and revoke API tokens. Terraform Cloud has three kinds of API tokens: user, team, and organization. Users can be members of multiple organizations, so user tokens work with any organization where the associated user is a member. Refer to [API Tokens](/cloud-docs/users-teams-organizations/api-tokens) for details.

API tokens are required for the following tasks:

- Authenticating with the [Terraform Cloud API](/cloud-docs/api-docs). API calls require an `Authorization: Bearer <TOKEN>` HTTP header.
- Authenticating with the [Terraform Cloud CLI integration](/cli/cloud/settings) or the [`remote` backend](/language/settings/backends/remote). These require a token in the CLI config file or in the backend configuration.

> - Using [private modules](/cloud-docs/registry/using) in command-line runs on local machines. This requires [a token in the CLI config file](/cloud-docs/registry/using#authentication).

Reviewer 3: ```suggestion
- Using [private modules](/cloud-docs/registry/using) in command-line runs on local machines. This requires [a token in the CLI configuration file](/cloud-docs/registry/using#authentication).
```

> Protect your tokens carefully because they can do anything your user account can. For example, if you belong to a team with permission to read and write variables for a workspace, your API token can edit variables in that workspace. Refer to [permissions](/cloud-docs/users-teams-organizations/permissions) for more details.

Reviewer 2: ```suggestion
Protect your tokens carefully because they contain the same permissions as your user account. For example, if you belong to a team with permission to read and write variables for a workspace, another user could use your API token to authenticate as your user account and also edit variables in that workspace. Refer to [permissions](/cloud-docs/users-teams-organizations/permissions) for more details.
```
I just want to rephrase this as another option which clicked a bit easier in my head, referring to the API token as a tool with permissions instead of it being the actor making the changes.

Author: this is much much better. Thank you! I phoned it in and didn't rewrite as much as I should have here. I appreciate you!
